src/components/Title/TitleTag/index.tsx

```diff
@@ -27,7 +27,12 @@ type Props = {
 
 const TitleTag = ({ db, mainType, dbLabel }: Props) => {
   const isEntry = mainType === 'entry';
-  const isInterPro = db && db.toLowerCase() === 'interpro';
+  let isInterPro = false;
```

Comment:
I'm a bit puzzle by the name of this variable, why `isInterPro`?

Reply:
The `TitleTag` component is the label/tag that is at the top of most pages, with the source and type of data being browsed.

_UniProtKB/TrEMBL protein_

![image](https://github.com/ProteinsWebTeam/interpro7-client/assets/2097501/fb71b1ee-89da-47b3-aade-02373aab0793)

_PIRSF entry_

![image](https://github.com/ProteinsWebTeam/interpro7-client/assets/2097501/b06b9c03-cab9-47f9-80d6-b971b36eb3f0)

_InterPro entry_

![image](https://github.com/ProteinsWebTeam/interpro7-client/assets/2097501/59954c3c-d13a-43f4-a560-02a6a6926e57)

The `isEntry` and `isInterPro` variables are used to:

* select the colour of the tag (`#1878ba` or `#4e7287`)
* generate the link to the correct section on the entry page on docs whether the entry is an InterPro entry or a member database signature.

Reply:
Thanks for clarifying. I was a bit confused as when I initially read the code, I thought you were checking whether the db was `interpro` (for all the entries and signatures) and thinking in my head that Pfam is also part of InterPro.
After looking at it again, I got what you are trying to do.